client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -215,6 +227,191 @@ public void onFailure(Exception e) {
         }
     }
 
+    public void testUpdate() throws IOException {
+        RestHighLevelClient client = highLevelClient();
+        {
+            IndexRequest indexRequest = new IndexRequest("index", "type", "id").source("field", 0);
+            IndexResponse indexResponse = client.index(indexRequest);
+            assertSame(indexResponse.status(), RestStatus.CREATED);
+
+            XContentType xContentType = XContentType.JSON;
+            BytesRef script = XContentBuilder.builder(xContentType.xContent())
+                    .startObject()
+                        .startObject("script")
+                            .field("lang", "painless")
+                            .field("code", "ctx._source.field += params.count")
+                        .endObject()
+                    .endObject().bytes().toBytesRef();
+            HttpEntity body = new ByteArrayEntity(script.bytes, script.offset, script.length, ContentType.create(xContentType.mediaType()));
+            Response response = client().performRequest(HttpPost.METHOD_NAME, "/_scripts/increment-field", emptyMap(), body);
+            assertEquals(response.getStatusLine().getStatusCode(), RestStatus.OK.getStatus());
+        }
+        {
+            //tag::update-request
+            UpdateRequest request = new UpdateRequest(
+                    "index", // <1>
+                    "type",  // <2>
+                    "id");   // <3>
+            //end::update-request
+            //tag::update-request-with-inline-script
+            Map<String, Object> parameters = new HashMap<>(); // <1>
+            parameters.put("count", 4);
+
+            Script inline = new Script(ScriptType.INLINE, "painless", "ctx._source.field += params.count", parameters);  // <2>
+            request.script(inline);  // <3>
+            //end::update-request-with-inline-script
+
+            if (randomBoolean()) {
+                //tag::update-request-with-stored-script
+                Script stored = new Script(ScriptType.STORED, "painless", "increment-field", parameters);  // <1>
```

Comment:
can you split this line in two? otherwise the <1> ends up in a new line in the rendered docs